engine/src/test/resources/testAppPackage/mydtapp/src/main/java/com/example/mydtapp/StdoutOperator.java

```diff
@@ -27,6 +27,19 @@
 
 public class StdoutOperator extends BaseOperator
 {
+
```

Comment:
for (int i = 0; i < operatorsJSONArray.length(); i++) {
 +      JSONObject operator = operatorsJSONArray.getJSONObject(i);
 +      if (operator.get("name").equals("stdout")) {
 +        Assert.assertEquals( "default-value-1", operator.getJSONObject("properties").get("testInput"));
 +        Assert.assertEquals( "com.example.mydtapp.StdoutOperator", operator.getJSONObject("properties").get("@class"));
 +      }
 +    }